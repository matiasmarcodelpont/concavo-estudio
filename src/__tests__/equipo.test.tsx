import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import Equipo from '@/app/equipo/page'

describe('Equipo', () => {
  it('renders equipo unchanged', () => {
    const { container } = render(<Equipo />)
    expect(container).toMatchSnapshot()
  })
})
