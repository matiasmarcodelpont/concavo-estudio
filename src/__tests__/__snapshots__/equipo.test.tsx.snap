// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Equipo renders equipo unchanged 1`] = `
<div>
  <main>
    <h1>
      Equipo
    </h1>
  </main>
</div>
`;
